cli/index.js

```diff
@@ -77,8 +85,15 @@ if (cli.flags.verbose) {
   flags.logLevel = 'error';
 }
 
+// Normalize audit whitelist.
+if (!flags.auditWhitelist || flags.auditWhitelist === 'all') {
+  flags.auditWhitelist = null;
+} else {
+  flags.auditWhitelist = new Set(flags.auditWhitelist.split(','));
```

Comment:
we don't have to. We could `log.warn` something about audit by that name doesn't exist?